scripts/code_style.py

```diff
@@ -75,8 +75,24 @@ def get_src_files(since: Optional[str]) -> List[str]:
     output = subprocess.check_output(["git", "ls-files"] + file_patterns,
                                      universal_newlines=True)
     src_files = output.split()
+
+    # When this script is called from a git hook, some environment variables
+    # are set by default which force all git commands to use the main repository
+    # (i.e. prevent us from performing commands on the framework repo).
+    # Create an environment without these variables for running commands on the
+    # framework repo.
+    framework_env = os.environ.copy()
```

Comment:
Just a comment that we may want to do that in check_files.py as well. Not necessarily in this PR though.